python/src/wrfcloud/aws/resources/cf_imagebuilder_wrf_intel.yaml

```
@@ -314,7 +314,7 @@ Resources:
                     - source /etc/bashrc
                     - mkdir -p /opt/src
                     - cd /opt/src
-                    - git clone -b feature/auto-install https://github.com/NCAR/wrfcloud
+                    - git clone -b feature/112-model-configs https://github.com/NCAR/wrfcloud
```

Should this be develop since this branch will go away when it is merged?
```suggestion
                    - git clone -b develop https://github.com/NCAR/wrfcloud
```

---

John is working on a fix for this in a separate branch.